build/actions.sh

```diff
@@ -12,6 +12,7 @@ set -o errexit
 source build/common.sh
 
 write-release-date() {
+  mkdir -p _build
```

Comment:
Right, but given the instructions on the Contributing page `make` won't have run the first time `write-release-date` does.